packages/postcss-svgo/src/index.js

```diff
@@ -7,6 +7,7 @@ import { encode, decode } from './lib/url';
 const PLUGIN = 'postcss-svgo';
 const dataURI = /data:image\/svg\+xml(;((charset=)?utf-8|base64))?,/i;
 const dataURIBase64 = /data:image\/svg\+xml;base64,/i;
+const base64String = /(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=|[A-Za-z0-9+/]{4})/;
```

Comment:
No need regexp, please use URL parser

Reply:
OK, did the changes, hopefully I understood what you meant correctly.